modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/configuration/admin/service/OpenIdConnectProviderManagedServiceFactory.java

```diff
@@ -475,31 +506,37 @@ private void _updateOAuthClientEntry(
 		}
 
 		try {
-			String authServerWellKnownURI = _updateOAuthClientASLocalMetadata(
-				defaultUserId, properties);
-
-			OAuthClientEntry oAuthClientEntry =
-				_oAuthClientEntryLocalService.fetchOAuthClientEntry(
-					companyId, authServerWellKnownURI,
-					_getPropertyAsString("openIdConnectClientId", properties));
-
-			if (oAuthClientEntry == null) {
-				oAuthClientEntry =
-					_oAuthClientEntryLocalService.addOAuthClientEntry(
-						defaultUserId,
-						_generateAuthRequestParametersJSON(properties),
-						authServerWellKnownURI, _generateInfoJSON(properties),
-						OAuthClientEntryConstants.OIDC_USER_INFO_MAPPER_JSON,
-						_generateTokenRequestParametersJSON(properties));
+			OAuthClientEntry oAuthClientEntry = null;
+
+			if (oldProperties != null) {
+				String oldAuthServerWellKnownURI =
+					_updateOAuthClientASLocalMetadata(
+						defaultUserId, properties);
```

Comment:
For me it's a bit strange to call the variable `oldAuthServerWellKnownURI` but using `properties` to get it. I would assume to use `oldProperties` or not call it `old`.